src/Woz.BevragenProxy.Specifications/BepaalRelevanteWaarden.feature

```diff
@@ -0,0 +1,180 @@
+
+    Voorbeelden:
+    | status_enum                  | statuscode | status                                                |
+    | beschikking_genomen          | 01         | beschikking genomen                                   |
+    | beschikking_herzien          | 03         | herzieningsbeschikking                                |
+    | bezwaar_gehandhaafd          | 11         | bezwaar afgehandeld, beschikking gehandhaafd          |
+    | bezwaar_veranderd            | 12         | bezwaar afgehandeld, vastgestelde waarde veranderd    |
+    | waarde_ambtshalve_verminderd | 13         | waardeambtshalveverminderd                            |
+    | beroep_gehandhaafd           | 21         | uitspraak beroep, beschikking gehandhaafd             |
+    | beroep_veranderd             | 22         | uitspraak beroep, vastgestelde waarde veranderd       |
+    | hoger_beroep_gehandhaafd     | 24         | uitspraak hoger beroep, beschikking gehandhaafd       |
+    | hoger_beroep_veranderd       | 25         | uitspraak hoger beroep, vastgestelde waarde veranderd |
+    | hoge_raad_gehandhaafd        | 31         | arrest Hoge Raad, beschikking gehandhaafd             |
+    | hoge_raad_veranderd          | 32         | arrest Hoge Raad, vastgestelde waarde veranderd       |
+    | hoge_raad_geding_verwezen    | 33         | arrestHogeRaad,gedingverwezen                         |
+    | voorlopige_aanslag           | 99         | waarde te gebruiken voor voorlopige aanslag           |
+
+Scenario: indicatieBezwaarBeroep wanneer er meerdere statussen zijn bij een waardepeildatum en ingangsdatum
+	Gegeven een WOZ-object bevat de volgende waarden
+	| vastgesteldeWaarde | waardepeildatum | ingangsdatum | beschikkingsStatussen                         |
+	| 437000             | 2019-01-01      | 2020-01-01   | beroep_gehandhaafd, beroep_aangetekend        |
+	| 392000             | 2018-01-01      | 2019-01-01   | hoger_beroep_aangetekend, beschikking_genomen |
+	Als de relevante waarden zijn bepaald voor het WOZ-object
+	Dan bevat het WOZ-object de volgende waarden
+	| vastgesteldeWaarde | waardepeildatum | indicatieBezwaarBeroep |
+	| 437000             | 2019-01-01      | true                   |
+	| 392000             | 2018-01-01      | true                   |
+
+Scenario: indicatieBezwaarBeroep wanneer er bezwaar, (hoger)beroep of cassatie loopt bij niet de meest recente ingangsdatum
+	Gegeven een WOZ-object bevat de volgende waarden
+	| vastgesteldeWaarde | waardepeildatum | ingangsdatum | beschikkingsStatussen |
+	| 215000             | 2018-01-01      | 2019-05-01   | beschikking_genomen   |
+	| 215000             | 2018-01-01      | 2019-01-01   | beroep_aangetekend    |
+	Als de relevante waarden zijn bepaald voor het WOZ-object
+	Dan bevat het WOZ-object de volgende waarden
+	| vastgesteldeWaarde | waardepeildatum |
+	| 215000             | 2018-01-01      |
```

Comment:
wil je hier niet expliciet controleren dat er geen indicatieBezwaarBeroep in de waarden zit?

Reply:
Toegevoegd. De scenario titels zijn ook aangepast om de situatie beter te beschrijven